### PR TITLE
✨ Change contact button to show crisp webview

### PR DESCRIPTION
**Pull request by williamchong007**
Removed intercom, and do not log events for now since I cannot figure out how to do it in background with a webview

### CODE REVIEW

File: app/screens/crisp-support-screen/crisp-support-screen.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  render() {
+    const { CRISP_WEBSITE_ID } = this.props
+    const { currentUser } = this.props.userStore
+    const { profile } = this.props.userStore.authCore
+    let uri = CRISP_WEBSITE_ID ? `https://go.crisp.chat/chat/embed/?website_id=${CRISP_WEBSITE_ID}` : 'https://help.like.co'
+    if (CRISP_WEBSITE_ID) {
+      if (currentUser && currentUser.email) uri += `&email=${encodeURIComponent(currentUser.email)}`
+      if (profile && profile.primaryPhone) uri += `&phone=${encodeURIComponent(profile.primaryPhone)}`
+    }
+    console.log(uri)
```
Comment (nwingt) [inline app/screens/crisp-support-screen/crisp-support-screen.tsx:43]: Please remove this log

File: app/screens/crisp-support-screen/crisp-support-screen.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+  render() {
+    const { CRISP_WEBSITE_ID } = this.props
+    const { currentUser } = this.props.userStore
+    const { profile } = this.props.userStore.authCore
+    let uri = CRISP_WEBSITE_ID ? `https://go.crisp.chat/chat/embed/?website_id=${CRISP_WEBSITE_ID}` : 'https://help.like.co'
+    if (CRISP_WEBSITE_ID) {
+      if (currentUser && currentUser.email) uri += `&email=${encodeURIComponent(currentUser.email)}`
+      if (profile && profile.primaryPhone) uri += `&phone=${encodeURIComponent(profile.primaryPhone)}`
+    }
```
Comment (nwingt) [inline app/screens/crisp-support-screen/crisp-support-screen.tsx:42]: I think u can add a view property in the `userStore` to return this URL, and remove the `CRISP_WEBSITE_ID` props as well

```javascript
...
get crispChatEmbedURL() {
   ...
   return url
}
...
```

File: app/navigation/app-navigator.tsx [hunk 3]
```diff
@@ -19,6 +19,7 @@ import { QrcodeScannerScreen } from "../screens/qrcode-scanner-screen"
 import { ReceiveScreen } from "../screens/receive-screen"
 import { StakingRewardsWithdrawScreen } from "../screens/staking-rewards-withdraw-screen"
 import { ValidatorScreen } from "../screens/validator-screen"
+import { CrispSupportScreen } from "../screens/crisp-support-screen"
```
Comment (nwingt) [inline app/navigation/app-navigator.tsx:22]: Please sort the import

File: app/navigation/app-navigator.tsx [hunk 4]
```diff
@@ -54,6 +55,7 @@ export const AppNavigator = createStackNavigator({
   StakingRewardsWithdraw: StakingRewardsWithdrawScreen,
   Transfer: TransferNavigator,
   Validator: ValidatorScreen,
+  CrispSupport: CrispSupportScreen,
```
Comment (nwingt) [inline app/navigation/app-navigator.tsx:58]: Please sort the object key

**Comment by nwingt** [comment]
@williamchong007  Please also clean up the linter warnings

File: app/screens/crisp-support-screen/crisp-support-screen.tsx (hunk 2 above)
Comment (williamchong007) [reply app/screens/crisp-support-screen/crisp-support-screen.tsx:42]: but crisp website id seems to have nth to do with user, do any other useful store exists?

Comment (nwingt) [reply app/screens/crisp-support-screen/crisp-support-screen.tsx:42]: As I saw the URL is depending on the user's email & the AuthCore's profile, so I suggest using computed property in MST to compose the URL, also make the `render()` looks cleaner

Comment (nwingt) [reply app/screens/crisp-support-screen/crisp-support-screen.tsx:42]: Get this straight
```tsx
// user-store.ts
...
get crsipChatEmbedURL() {
  const crispID = self.env.appConfig.getValue("CRISP_WEBSITE_ID")
  if (crispID) {
    const { profile } = self.authCore
    let uri = `https://go.crisp.chat/chat/embed/?website_id=${crispID}`
    if (self.currentUser?.email) {
      uri += `&email=${encodeURIComponent(self.currentUser.email)}`
    }
    if (profile?.primaryPhone) {
      uri += `&phone=${encodeURIComponent(profile.primaryPhone)}`
    }
  }
  return "https://help.like.co"
}
...

// crisp-support-screen.tsx
<WebView source={{ uri: this.props.userStore.crsipChatEmbedURL }} {...} />
```

Comment (nwingt) [reply app/screens/crisp-support-screen/crisp-support-screen.tsx:42]: Get this straight 
```tsx
// user-store.ts
...
get crispChatEmbeddedURL() {
  const crispID = self.env.appConfig.getValue("CRISP_WEBSITE_ID")
  let uri: string
  if (crispID) {
    const { profile } = self.authCore
    uri = `https://go.crisp.chat/chat/embed/?website_id=${crispID}`
    if (self.currentUser?.email) {
      uri += `&email=${encodeURIComponent(self.currentUser.email)}`
    }
    if (profile?.primaryPhone) {
      uri += `&phone=${encodeURIComponent(profile.primaryPhone)}`
    }
  } else {
    uri = "https://help.like.co"
  } 
  return uri
}
...

// crisp-support-screen.tsx
<WebView source={{ uri: this.props.userStore.crispChatEmbeddedURL }} {...} />
```

**Pull request merged**